LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Crown Copyright (Government Digital Service)
```

Comment:
Fair enough! I won’t object to leaving it as just the year of the earliest code.